editor/editor_data.cpp

```diff
@@ -719,6 +719,18 @@ bool EditorData::check_and_update_scene(int p_idx) {
 				new_selection.push_back(new_node);
 			}
 		}
+		if (pscene.is_valid()) {
+			//Apply Overrides when instanced scene gets updated
+			auto o = pscene->get_state()->get_overrides();
```

Comment:
```suggestion
			Vector<SceneState::NodeData> o = pscene->get_state()->get_overrides();
```
`auto` is [not allowed](https://docs.godotengine.org/en/stable/contributing/development/cpp_usage_guidelines.html#auto-keyword)